steps/dns/steps.go

```diff
@@ -98,7 +106,8 @@ func parseOptionsTable(ctx context.Context, t *godog.Table) ([]dns.EDNS0, error)
 	for i, o := range options {
 		data, err := hex.DecodeString(o.Data)
 		if err != nil {
-			return nil, fmt.Errorf("Error converting to byte array: %s. %s", o.Data, err)
+			//return nil, fmt.Errorf("Error converting to byte array: %s. %s", o.Data, err)
+			data = []byte(o.Data)
```

Comment:
Why do you do that?

Reply:
I'd like to remove this restriction to allow any value in OPT records. Currently only SHA256 values are allowed. F.I:

   ```
   And the DNS query options
          | code | data                 |
          | 1001 | Incorrect-Value |
          | 1002 | [EMPTY]           |
```